src/main/resources/assets/minestuck/models/block/steep_green_stone_brick_stairs_base.json

```diff
@@ -0,0 +1,113 @@
+{
+    "credit": "crafted with love by the grease artisan mysterymeat",
+    "texture_size": [64, 64],
+    "textures": {
+        "2": "minestuck:block/steep_green_stone_brick_stairs_base",
```

Comment:
I'm pretty sure the texture could be split up and reorganized to clear out blank space in the texture.
Also, "2" is not a great texture name

Reply:
Each cube uses several different texture types and I dont see a way to apply specifically one texture file to only one of the faces of a cube

Reply:
Texture names are now "texture" instead of 2

Reply:
It for sure is possible to apply different textures for different facings within the json format (though I don't know what limitations blockbench might have)
I suppose it's not too big of a deal, so I am fine if you decide to leave it be

Reply:
I would be fine leaving it